File: uri/1 - Basics/1013/Main.java
import java.util.*;

public class Main{

    public static void main(String args[]){
        Scanner scanner = new Scanner(System.in);

        int a = scanner.nextInt();
        int b = scanner.nextInt();
        int c = scanner.nextInt();
        int aux=0;
        
        if(a > b)
            aux = a;
        else
            aux = b;
        
        if(aux > c)
            aux = aux;
        else
            aux = c;
        
        System.out.format("%d eh o maior\n", aux);
    }
    
}

File: uri/1 - Basics/1013/Main.go
package main

import (
	"fmt"
)

func main() {
	var a, b, c, aux int

	fmt.Scanf("%d %d %d", &a, &b, &c)

	if a > b {
		aux = a
	} else {
		aux = b
	}

	if aux < c {
		aux = c
	}

	fmt.Printf("%d eh o maior\n", aux)

}
